### PR TITLE
Add Blitz.app latest

### PR DESCRIPTION
**Pull request by 1337MARCEL**
Added cask for Blitz app.

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.
- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not [already refused].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/blitz.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask 'blitz' do
+  version '0.9.16'
+  sha256 'fbc5a899ca22bbadd2751997d31bf4aa345d1b6e0e9d604dcfdd1ad16621c2ec'
+
+  url 'https://dl.blitz.gg/download/mac'
+  appcast 'https://www.corecode.io/cgi-bin/check_urls/check_url_filename.cgi?url=https://dl.blitz.gg/download/mac'
```
Comment (vitorgalvao) [inline Casks/blitz.rb]: @core-code Insights as to why it doesn’t work in this case?

Comment (core-code) [reply Casks/blitz.rb]: not really, i'll have a look later today.

Comment (core-code) [reply Casks/blitz.rb]: @vitorgalvao: the server returned '403 forbidden' for requests done with the default python 'user-agent'. i've now changed the default 'user-agent' to 'Mozilla' which makes this work now. furthermore you can send an additional parameter 'user_agent' to the CGI to customise the used user-agent

Comment (vitorgalvao) [reply Casks/blitz.rb]: @core-code Thank you.

**Comment by vitorgalvao** [comment]
Thank you @1337MARCEL, and welcome.

**Pull request merged**